src/screens/SUE/SueListScreen.tsx

```diff
@@ -320,6 +319,7 @@ export const SueListScreen = ({ navigation, route }: Props) => {
                       placeholder: texts.sue.filter.alleSortingTypes
                     }
                   ]}
+                  initialStartDate={initial_start_date.start_date}
```

Comment:
why do we pass that as prop. it is always the same value and just for the case to identify the initial date right? we can have it fix in the other file or export the variable from the top here. this is nothing we need to make the component aware of per props.

Reply:
Yes, that’s correct. This is just a fixed date. But we use the Filter component in many places, and we only need to consider `start_date` in this specific case. If we add `start_date` as a fixed value to the entire Filter component, I think it might break other instances of the Filter component.

Reply:
moving `export const initialStartDate = { start_date: '1900-01-01T00:00:00+01:00' };` in the other file should not brake anything

Reply:
in which cases you do not want an initial start date?

Reply:
Only for SUE Module

Reply:
I tested it again. using export/import instead of prop does not cause any problem. 

[8b97b8c](https://github.com/smart-village-solutions/smart-village-app-app/pull/1252/commits/8b97b8c8c9eef7c0d4bcb33854c0c682b5fbe646)